### PR TITLE
Add propTypes and jsDoc for components in Pages

### PR DESCRIPTION
**Pull request by guillermoparral1995**
Issue: #143

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,6 +76,7 @@
     "better-docs": "^1.4.7",
     "husky": "^4.2.3",
     "jest-dom": "^3.5.0",
+    "jsdoc": "^3.6.4",
```
Comment (qdozaq) [inline package.json:79]: Thats strange. This is already on master. Can you rebase?

File: src/app/component/MapView/MapView.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { MapStyle } from "./MapView.style";
 // import Routing from './MapView-routing'
 
 function MapView(props) {
-  console.log(props)
+  console.log(props);
```
Comment (qdozaq) [inline src/app/component/MapView/MapView.js:7]: This should just be removed

File: package.json (hunk 1 above)
Comment (qdozaq) [reply package.json:79]: And remove the changes from the package-json.lock

**Comment by azmainamin** [comment]
There are also multiple conflicts that need to be resolved

File: package.json (hunk 1 above)
Comment (guillermoparral1995) [reply package.json:79]: i think the problem is that I used it as a dev dependency but it was already installed as non-dev dependency. i think it should remain dev

Comment (qdozaq) [reply package.json:79]: Oh I thought this had be moved to dev. Yeh you can leave this one and remove the one from `dependencies` that'd be great.

File: package.json [hunk 3]
```diff
@@ -15,7 +15,6 @@
     "date-fns": "^2.12.0",
     "express": "^4.17.1",
     "firebase": "^7.14.0",
-    "jsdoc": "^3.6.4",
```
Comment (qdozaq) [inline package.json:18]: Looks like this didn't get moved to `dev-dependencies`

Comment (guillermoparral1995) [reply package.json:18]: whoops, now it should be ok! 😅

**Pull request merged**